src/realm/sync/noinst/client_impl_base.cpp

```diff
@@ -369,7 +377,7 @@ void Connection::cancel_reconnect_delay()
         // reset back to false, because in that case, we know that the
```

Comment:
some part of this comment is not accurate anymore

Reply:
I guess it's still correct, but `m_scheduled_reset` is not exposed anymore so I thought it should not be mentioned